ui/packages/ui/webpack.common.js

```diff
@@ -3,10 +3,12 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 const webpack = require('webpack');
 const { dependencies } = require("./package.json");
+const GitRevisionPlugin = require('git-revision-webpack-plugin');
+const gitRevisionPlugin = new GitRevisionPlugin({commithashCommand: 'rev-parse --short HEAD'});
 
 // Try the environment variable, otherwise use root
 const ASSET_PATH = process.env.ASSET_PATH || '/';
-const COMMIT_HASH = process.env.COMMIT_HASH || require('child_process').execSync('git rev-parse --short HEAD').toString().trim();
+const COMMIT_HASH = process.env.COMMIT_HASH || gitRevisionPlugin.commithash();
```

Comment:
```suggestion
const COMMIT_HASH = process.env.COMMIT_HASH || gitRevisionPlugin.version();
```

Reply:
We can capture the latest tag (that build is based on, plus the short hash) like this:
```
$ git describe --always --tags
v1.5.0.Beta2-2-g3cf30f2
```

Reply:
What happens here, when git is not available? Did you test that (removing `git` from the `microdnf install git` call in the builder part of the Dockerfile? Do we have a proper error message?

Reply:
It looks like we need `git` installed (see https://github.com/pirelenito/git-revision-webpack-plugin/blob/v3/lib/helpers/run-git-command.js#L9)

Reply:
I know. But the question is what happens when it's not available. I expect the build to fail, and not, like before, continue without assets.

Reply:
I tried after removing `git` from  `Dockerfile` and build was success this time.

Reply:
And what is the COMMIT_HASH then or is it just empty? This should not happen. When the hash (and git) is unavailable the build should fail. WDYT?

Reply:
Yes, the build should fail when `git` is unavailable but build is success and I am getting`COMMIT_HASH` is `3762ffd`.

Reply:
@uidoyen So what is your local `git rev-parse --short HEAD`? Is it the correct value? And why does it "work"?